src/containers/shared/components/CopyToClipboard.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import copy from '../images/copy.png'
+
+interface Props {
+  text: string
+  className?: string
+}
+
+const CopyToClipboard = ({ text, className }: Props) => (
```

Comment:
Please add a test for this component.  Also this one could easily be its own pull request to not have it mired in this one.

Reply:
would we need to use something like test-library then? I dont know if we can mock something like this with wrapper.